SU-SWS/linky_clicky
# Stanford Sites Helper refactor

# READY FOR REVIEW

# Summary
- Cuts down the running time of the `stanford_sites_helper.feature` by about 80%

# Needed By (Date)
- Whenevah

# Criticality
- How critical is this PR on a 1-10 scale? 2
- But it does help get our Travis runs under the 49 minute threshold

# Steps to Test

1. Check out this branch
2. Run `sites/uat/features/high-value/stanford_sites_helper.feature`
3. Ensure that it passes
4. Review the diff to see that nothing was missed

# Affected Projects or Products
- https://github.com/SU-SWS/Stanford-Drupal-Profile

# Associated Issues and/or People
- Anyone who should be notified? ( @kbrownell )
